realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/AuthTests.java

```diff
@@ -447,6 +446,21 @@ public void singleUserCanBeLoggedInAndOutRepeatedly() {
         }
     }
 
+    // verify that multiple users can be logged in at the same time
```

Comment:
I don't quite get the point of this UT. From the description, it would be the same with `multipleUsersCanBeLoggedInSimultaneously`. But from the implementation, it tries to test login-logout with different users? Maybe change the description of the test?